src/liquidex.rs

```diff
@@ -39,59 +30,141 @@ impl LiquidexMakeOpt {
     }
 }
 
-// Clone of TxOutSecrets, but with the name changed to match the previous struct.
-// This is a temporary solution since soon we should be able to migrate to PSET.
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
-pub struct LiquidexTxOutSecrets {
-    asset: elements::AssetId,
-    asset_blinder: elements::confidential::AssetBlindingFactor,
-    amount: u64,
-    amount_blinder: elements::confidential::ValueBlindingFactor,
-}
-
-impl LiquidexTxOutSecrets {
-    pub fn to_txoutsecrets(&self) -> elements::TxOutSecrets {
-        elements::TxOutSecrets {
-            asset: self.asset,
-            asset_bf: self.asset_blinder,
-            value: self.amount,
-            value_bf: self.amount_blinder,
-        }
-    }
+pub struct LiquidexV1TxOutSecrets {
```

Comment:
In the post you say you are doing a breaking changes (and it's fine because I am reasonably sure there are no impl in the wild), so I would name this without the `V1` as before, and also add a field version to it so it will become easier for future version

Reply:
done